src/components/ebay-icon/index.js

```diff
@@ -0,0 +1,39 @@
+const markoWidgets = require('marko-widgets');
+const processHtmlAttributes = require('../../common/html-attributes');
+const template = require('./template.marko');
+
+function getTemplateData(state, input) {
+    const type = input.type || 'background';
+    const isBackground = type === 'background';
+    const isInline = type === 'inline';
+    const accessibilityText = input.accessibilityText;
+    const name = input.name;
+    let titleId;
+    let accessibilityAttributes;
+
+    if (isInline) {
+        if (accessibilityText) {
+            titleId = `icon-title-${Math.random().toString(36).substr(2, 9)}`;
+            accessibilityAttributes = { 'aria-labelled-by': titleId, focusable: 'true', role: 'img' };
```

Comment:
I think we should leave focusable as false. If it's an interactive image it should be nested inside a button or link (which will take care of focus) by app team.